src/SDKs/PowerBIDedicated/Management.PowerBIDedicated/Microsoft.Azure.Management.PowerBIDedicated.csproj

```diff
@@ -0,0 +1,44 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <Description>Provides management capabilities for Azure PowerBI Dedicated accounts.</Description>
+    <AssemblyTitle>Microsoft Azure PowerBI Dedicated Management Library</AssemblyTitle>
+    <VersionPrefix>2.0.1</VersionPrefix>
+    <AssemblyName>Microsoft.Azure.Management.PowerBIDedicated</AssemblyName>
+    <PackageId>Microsoft.Azure.Management.PowerBIDedicated</PackageId>
+    <PackageTags>Microsoft Azure resource management;asazure;REST HTTP client;azureofficial;windowsazureofficial;netcore451511</PackageTags>
+  </PropertyGroup>
+  <PropertyGroup>
+    <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net452' ">
+    <Reference Include="System" />
+    <Reference Include="Microsoft.CSharp" />
+  </ItemGroup>
+
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.4' ">
```

Comment:
@selasagi you don't need any of these references. Please model your sdk project file by looking at other project files in the repo, e.g. https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/BatchAI/Management.BatchAI/Microsoft.Azure.Management.BatchAI.csproj